packages/router/src/__typetests__/tsconfig.json

```diff
@@ -1,10 +1,12 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "composite": false,
     "incremental": true,
-    "declaration": false,
     "declarationMap": false,
-    "emitDeclarationOnly": false
+    "emitDeclarationOnly": false,
+    "types": []
   },
+  "include": ["./"],
+  "exclude": [],
+  "references": [{"path": "../../"}]
```

Comment:
`composite` was a problem before, but it works as expected with TSTyche. `include`, `exclude`, `references` must be here too, because root TSConfig excludes `**/*.test.*`.